apps/v4/registry/new-york-v4/ui/sidebar/SidebarProvider.vue

```diff
@@ -11,7 +11,7 @@ const props = withDefaults(defineProps<{
   open?: boolean
   class?: HTMLAttributes["class"]
 }>(), {
-  defaultOpen: true,
+  defaultOpen: !document.cookie.includes(`${SIDEBAR_COOKIE_NAME}=false`),
```

Comment:
Hi, Thanks for the PR ❤️ 

How about using `defaultDocument` from `@vueuse/core`?  for preventing errors on the server-side 


```suggestion
  defaultOpen: defaultDocument?.cookie.includes(`${SIDEBAR_COOKIE_NAME}=false`) ?? true,
```

Reply:
@sadeghbarati great idea! I updated to this:
```
defaultOpen: !defaultDocument?.cookie.includes(`${SIDEBAR_COOKIE_NAME}=false`),
```

combined our solutions with `defaultDocument` and keeping the `!` for the open state to work correctly

Thanks!